src/main/java/com/recruitment/project/parking/parkingspaces/controller/UserVisitParkingController.java

```diff
@@ -0,0 +1,62 @@
+package com.recruitment.project.parking.parkingspaces.controller;
+
+import com.recruitment.project.parking.parkingspaces.domain.UserParkingVisit;
+import com.recruitment.project.parking.parkingspaces.service.UserParkingVisitService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.format.annotation.DateTimeFormat;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.List;
+
+@CrossOrigin(origins = "*", maxAge = 3600)
+@RestController
+@RequestMapping("/parking")
+public class UserVisitParkingController {
+
+    private UserParkingVisitService userParkingVisitService;
+
+    @Autowired
+    public UserVisitParkingController(UserParkingVisitService userParkingVisitService) {
+        this.userParkingVisitService = userParkingVisitService;
+    }
+
+    @RequestMapping(value = "/start/{registration}/{isDisabled}", method = RequestMethod.GET)
+    public ResponseEntity<?> startParkingVisit(@PathVariable("registration") String registrationNumber, @PathVariable("isDisabled") boolean isDisabled) {
+        userParkingVisitService.startUserParkingVisit(registrationNumber, isDisabled,null, null);
+        return ResponseEntity.ok().body("Started");
```

Comment:
Zwracanie odpowiedzi stringiem to nienajlepszy pomysł.

Reply:
Tutaj powinna być metoda POST. Ale faktycznie jeżeli już chciałem coś takiego zwrócić to sam status   byłby wystarczający.